packages/edit-site/src/components/editor/site-preview.js

```diff
@@ -13,10 +14,12 @@ export default function SitePreview() {
 		return siteData?.home;
 	}, [] );
 
-	// If theme is block based, return the Editor, otherwise return the site preview.
```

Comment:
P.S. I removed this comment as I don't think it represents an actual implementation. The `SitePreview` component is [not used in block-based themes](https://github.com/WordPress/gutenberg/blob/779d04193d19ac3e750b3d146d41da3f28fe265b/packages/edit-site/src/components/editor/index.js#L251-L252).